### DISCUSSION TITLE
Adding the footer to the 404 page.

### DISCUSSION THREAD

**Discussion by Steve0Greatness** [theforumhelpers/theforumhelpers.github.io#211] [Ideas]
so how you could do it is with a `<p>` tag, then add the style attribute and make the padding on the top be 225px:
`<p style="padding-top: 225px;">`

**Comment by BarelySmooth** [comment]
Yes, but why a ```<p>``` tag? Wont a ```div``` be enough? (with the css located in ```resources/stylesheet.css```)

**Reply by Steve0Greatness** [reply]
How would it be stored there? Would it be a class? Or be the general `<div>` tag?

**Reply by leahcimto** [reply]
If this was to be implemented, you would use `<div class="lostFooter">` and have the CSS in the stylesheet (no inline css is used in the whole site)

**Comment by Accio1** [comment]
The footer isn't even needed on the 404 page though. The whole point is that you are clicking to get back to the homepage. Plus, 225px wouldn't work for devices with different sized screens.
